editor/src/clj/editor/gui.clj

```diff
@@ -835,9 +835,9 @@
 (defn- add-size-to-overridden-fields [overridden-fields]
   (vec (sort (conj overridden-fields size-prop-index))))
 
-(defn- strip-size-from-node-desc [node-desc]
+(defn- strip-size-from-shape-base-node-msg [node-desc]
   (-> node-desc
-      (assoc :size [0.0 0.0 0.0 0.0])
+      (assoc :size [200.0 100.0 0.0 1.0]) ; Default size for ShapeNode. The runtime uses this size if there is no texture applied.
```

Comment:
The editor is fine with stripping out the `:size`, but it seems we still need to write it to the files for Bob and the runtime.